rust/crd/src/lib.rs

```diff
@@ -863,195 +895,9 @@ pub struct NodeSelector {
 
 impl Atomic for NodeSelector {}
```

Comment:
What does the NodeSelector do? I can't seem to find any references to it in the code. Also, why is it Atomic?

Reply:
It's a left over. The functionality is now covered by the affinites.